### ISSUE TITLE
Outputting only --stats

### ISSUE THREAD

**Issue by minusf** [Genivia/ugrep#212]
Hello. Is there a way to output **only** the `--stats`?

The use case is when searching in tens of thousands of files and not really caring about file names, line numbers, etc only how many files matched or not matched.  The `--quiet` flag did something entirely different :}

**Comment by genivia-inc** [comment]
Good question. The `--quiet` flag only reports success/failure as an exit code, like GNU grep for batch use in shells as conditionals. This is the same as redirecting the output to /dev/null, since only the exit code is expected to be used.

There is no option to output `--stats` without results, but you could use `--format` to basically output nothing for each match with `--format=""`.

**Comment by genivia-inc** [comment]
PS. `--format` counts every match rather than counting lines with matches, so `--stats` is number of lines differs. Maybe `--stats` should be adjusted when used with `--format` in a future release.

**Comment by minusf** [comment]
`--format=""` works perfectly for my use case, thank you.

i am not sure i understand your comment regarding the difference in the counting, isn't `--format="%f:%m%~" --stats` giving the same output as `-c --stats` sans the terminal colours?

in any case, for this use case those numbers are not interesting anyway :}

ps. those `""` in the `Lines matched if:` are hard to parse for me :}

**Comment by genivia-inc** [comment]
> i am not sure i understand your comment regarding the difference in the counting, isn't `--format="%f:%m%~" --stats` giving the same output as `-c --stats` sans the terminal colours?

Yes, that is correct. But the `--stats` report counts matches as if `-o` or `-u` was used, i.e. every match counts. Not the lines with matches are counted, which is fewer if lines have more than one match.
